src/test/java/com/joss/conductor/mobile/LocomotiveTest.java

```diff
@@ -67,6 +67,7 @@ public void test_building_android_capabilities() {
         capabilities.setCapability(MobileCapabilityType.FULL_RESET, true);
         capabilities.setCapability(MobileCapabilityType.PLATFORM_VERSION, "6.0");
         capabilities.setCapability(AndroidMobileCapabilityType.AVD, "Nexus 13");
+        capabilities.setCapability(AndroidMobileCapabilityType.AVD_ARGS, "-no-window");
```

Comment:
Consider providing a comment on the specifics of adding multiple parameters. (spaces, or comma separated, or whatever syntax is needed) 

Adding a test for this would also help to show this off

Reply:
Regarding tests, given that it's a JSON object being created and accepting whatever strings specified for validation, I didn't see any particular value for making one for multiple avdArg parameters.

The syntax requirements would be the same as Appium expects, but I see your point.  I'll make updates to the readme as well before I merge this.